tensorflow/core/api_def/base_api/api_def_Asin.pbtxt

```diff
@@ -1,4 +1,4 @@
 op {
   graph_op_name: "Asin"
-  summary: "Computes asin of x element-wise."
+  summary: "Computes `tf.math.asin` of `x` element-wise."
 }
```

Comment:
You could say `asin`, but not `tf.math.asin`. That is a name specific to the Python API, the actual API name may be different in other languages (or even aliased in Python). 

If you feel you want to change it for Python only, you'll have to do this in the python_api path (which is an override, it won't exist yet for each op), but I'd discourage to specializing docstrings for each language.